CRM/Activity/BAO/Activity.php

```diff
@@ -312,9 +312,9 @@ public static function create(&$params) {
     }
 
     // Set priority to Normal for Auto-populated activities (for Cases)
-    if (CRM_Utils_Array::value('priority_id', $params) === NULL &&
+    if (!isset($params['priority_id']) &&
```

Comment:
OK - it's not the same but I guess priority_id can't be 0....

Reply:
How is this not the same @eileenmcnaughton? The default return value of `CRM_Utils_Array::value` is NULL if the variable isn't set, and `isset($foo = NULL)` returns FALSE.

Reply:
Right - it could be set & equal to NULL or not set - I think here it doesn't matter in that set & equal to null would still be empty